torch/nn/functional.py

```diff
@@ -1960,11 +1960,7 @@ def pairwise_distance(x1, x2, p=2, eps=1e-6):
         >>> output = F.pairwise_distance(input1, input2, p=2)
```

Comment:
GitHub won't let me tag the like I want, but... in the docs for pairwise_distance you should change the output shape to `(N,)`.